src/components/alert/alert.scss

```diff
@@ -149,3 +149,10 @@
     transform: scale(scale.$effect-shrink);
   }
 }
+
+.c-alert__error {
+  .c-alert__extra {
+    background-color: color.$status-error-light;
+    color: color.$status-error-dark;
+  }
+}
```

Comment:
This could be simplified by using CSS custom properties instead of hard-set colors. This would also allow them to vary per theme.